pom.xml

```diff
@@ -142,7 +142,7 @@
         <version.org.glassfish.javax.el>3.0.1-b08-jbossorg-1</version.org.glassfish.javax.el>
         <version.org.glassfish.javax.enterprise.concurrent>1.0</version.org.glassfish.javax.enterprise.concurrent>
         <version.org.glassfish.javax.json>1.0.3</version.org.glassfish.javax.json>
-        <version.org.hibernate>5.0.9.Final</version.org.hibernate>
+        <version.org.hibernate>5.1.0.Final</version.org.hibernate>
```

Comment:
Could you try a temporary change to use "5.1.1-SNAPSHOT", instead of "5.1.0.Final"?  This will give us an idea of whether some of the test failures are fixed already.